meta-mender-core/recipes-mender/mender-client/mender-client.inc

```diff
@@ -25,27 +25,42 @@ inherit go
 
 inherit systemd
 
-SYSTEMD_SERVICE_${PN} = "mender.service"
-FILES_${PN} += "${systemd_unitdir}/system/mender.service \
+SYSTEMD_SERVICE_${PN} = "${MENDER_CLIENT}.service"
```

Comment:
Question: I'm not sure if `MENDER_CLIENT` is the best name. Other names I can think of grow pretty long though. Like `MENDER_CLIENT_RECIPE_NAME`? Soo... Hmmm

Reply:
`MENDER_CLIENT` sounds good to me.